blobby/client.py

```diff
@@ -1,82 +0,0 @@
-from functools import partial
-from typing import Optional
-
-import boto3.session
-import botocore.exceptions
-from boto3.resources.base import ServiceResource
-
-from . import blob
-
-
-class StorageError(Exception):
-    pass
-
-
-class StorageClient:
-    chunk_size = 1024
-
-    def __init__(self,
-                 store_name: str,
-                 endpoint_url: str,
-                 s3: Optional[ServiceResource] = None) -> None:
-        self.session = boto3.session.Session()
-        if s3 is None:
-            self.s3 = self.session.resource(
-                's3', endpoint_url=endpoint_url
-            )  # pragma: no cover
-        else:
-            self.s3 = s3
-
-        self.bucket = self.s3.Bucket(store_name)
-
-    def get(self, key: str) -> bytes:
-        """Get the contents of an object.
-
-        Use get_reader to retrieve an object with more flexibility for
-        retrieving objects.
-
-        Arguments:
-            key (str): key of the object to look up in the state store
-        Returns: bytes
-            contents of the object
-        """
-        try:
-            return self.create_reader(key).read()
-        except botocore.exceptions.ClientError as e:
-            raise StorageError(f'Unable to download {key}', *e.args) from None
-
-    def create_reader(self, key: str) -> blob.BlobReader:
-        """Return an object with various methods of reading the blob.
```

Comment:
'various methods of reading the blob' is somewhat superfluous here (the return type clearly shows this). Perhaps something like:
`Create a BlobReader to read key from the configured S3 bucket`